presto-connector-tests/src/test/java/com/facebook/presto/tpch/TestTpchMetadata.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.tpch;
+
+import com.facebook.presto.connector.meta.SupportedFeatures;
+import com.facebook.presto.connector.unittest.BaseMetadataTest;
+import com.facebook.presto.connector.unittest.MetadataSchemaTest;
+import com.facebook.presto.connector.unittest.MetadataTableTest;
+import com.facebook.presto.spi.SchemaTableName;
+import com.facebook.presto.spi.connector.Connector;
+import com.facebook.presto.spi.connector.ConnectorFactory;
+import com.facebook.presto.testing.TestingConnectorContext;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+
+import java.util.List;
+import java.util.Map;
+
+import static com.google.common.collect.ImmutableList.toImmutableList;
+import static com.google.common.collect.Iterables.getOnlyElement;
+
+/*
+ * Don't do this for other connectors. The TPCH connector is special because
```

Comment:
Explicitly say what is `this`.
My understanding is `Don't add tests for other specific connectors to presto-connector-tests module`